code/game/objects/items/stacks/wrap.dm

```diff
@@ -86,7 +89,7 @@
 			I.forceMove(P)
 			var/size = round(I.w_class)
 			P.name = "[weightclass2text(size)] parcel"
-			P.w_class = size
+			P.w_class = max(I.w_class + (1 / 3), WEIGHT_CLASS_HUGE)
```

Comment:
Non-integer `w_class`? I don't want to say that this would break shit, but this _would_ break shit.